farg/core/ui/gui/central_pane.py

```diff
@@ -23,96 +23,100 @@
-
-    Args:
-      view_class1: A subclass of :py:class:`~farg.core.ui.gui.views.viewport.ViewPort`.
-      view_class2: Another subclass of :py:class:`~farg.core.ui.gui.views.viewport.ViewPort`.
-    """
-    self.viewports = [view_class1(self, 0, 0, self.width, self.height / 2 - 2),
-                      view_class2(self, 0, self.height / 2 + 2,
-                                  self.width, self.height / 2 - 2)]
-    self.ReDraw()
-
-  def SetThreeWaySplit(self, view_class1, view_class2, view_class3):
-    """Set central pane to contain a three view, one on top and two on the bottom.
-
-    Args:
-      view_class1: A subclass of :py:class:`~farg.core.ui.gui.views.viewport.ViewPort`.
-      view_class2: Another subclass of :py:class:`~farg.core.ui.gui.views.viewport.ViewPort`.
-      view_class3: Also a subclass of :py:class:`~farg.core.ui.gui.views.viewport.ViewPort`.
-    """
-    self.viewports = [view_class1(self, 0, 0, self.width, self.height / 2 - 2),
-                      view_class2(self, 0, self.height / 2 + 2,
-                                  self.width / 2 - 2, self.height / 2 - 2),
-                      view_class3(self, self.width / 2 + 2, self.height / 2 + 2,
-                                  self.width / 2 - 2, self.height / 2 - 2)]
-    self.ReDraw()
-
-  def SetNamedView(self, name):
-    """Set the named view by calling the function shored in named_views."""
-    if not name in self.named_views:
-      print('Unrecognized view %s' % name)
-      print('Available views: ', list(self.named_views.keys()))
-      sys.exit(1)
-    self.named_views[name](self)
-
-  def NamedViewCmd(self, name):
-    return (lambda: self.SetNamedView(name))
-
-  def SetupMenu(self, parent):
-    """Create menu for the central pane."""
-    menubar = Menu(self)
-
-    view_menu = Menu(menubar, tearoff=0)
-    for name in self.named_views:
-      view_menu.add_command(label=name,
-                            command=self.NamedViewCmd(name))
-    menubar.add_cascade(label='View', menu=view_menu)
-
-    debug_menu = Menu(menubar, tearoff=0)
-    debug_menu.add_command(label='Debug Relations',
-                           command=self.controller.workspace.DebugRelations)
-    menubar.add_cascade(label='Debug', menu=debug_menu)
-    parent.config(menu=menubar)
+	"""The central pane of the Tk-based UI. This can hold several displays."""
+
+	#: A dictionary from names to functions that sets views. A view is made up of a set of
+	#: viewports, each an instance of :py:class:`~farg.core.ui.gui.views.viewport.ViewPort`.
+	#: Methods are available in this class to create views (e.g., SetFullView and
+	#: SetVerticallySplitView). See farg.apps.seqsee.gui.gui.py for an example.
+	named_views = {}  # Not a constant. pylint: disable=C6409
+
+	#: Name of initial view.
+	default_initial_view = ''  # Not a constant. pylint: disable=C6409
+
+	def __init__(self, master, controller, *, height, width, background):
+		self.height = height
```

Comment:
And the indentation in this class. Are you following a particular scheme? Can we stick to indenting two spaces (or 4, if you prefer that).